alphalens/utils.py

```diff
@@ -964,3 +1017,168 @@ def diff_custom_calendar_timedeltas(start, end, freq):
     timediff = end - start
     delta_days = timediff.components.days - actual_days
     return timediff - pd.Timedelta(days=delta_days)
+
+def subportfolio_cumulative_returns(returns, period, freq=None):
+    """
```

Comment:
This function can be totally dropped in my opinion. Without a tear.* function calling this, I can't imagine users being able to make any good use of this functionality.

Reply:
Ok, I will remove it.